src/core/types/State.ts

```diff
@@ -113,14 +113,14 @@ export interface CreditLineState {
   user: {
     linePositions: IdToCreditPositionMap;
     lineAllowances: { [line: string]: { [token: string]: Integer } };
-    borrowerPositions: IdToCreditPositionMap;
+    portfolio: GetUserPortfolioResponse;
```

Comment:
We dont want it to be the response we want it to be the formatted data that we will actually use in that app

Reply:
in fact we dont even want to store the data here at all. We want to store all the data in `state.lines` and only hold references to lines/positions owned by user here

Reply:
@kibagateaux I'm confused. Are you saying you want `portfolio` object to be moved 1 level up in the `CreditLineState` type?

Like this?


```
export interface CreditLineState {
  ...
  userPortfolio: GetUserPortfolioResponse;
  user: {
    linePositions: IdToCreditPositionMap;
    lineAllowances: { [line: string]: { [token: string]: Integer } };
  };
  ...
  };
}

```